packages/browser/src/core/user/index.ts

```diff
@@ -398,3 +398,53 @@ export class Group extends User {
     return undefined
   }
 }
+
+export class UniversalStorage {
+  private cookies: Store
+  private localStorage: Store
+  private mem: Store
+
+  options: UserOptions = {}
+
+  constructor(isDisabled: boolean, cookieOptions?: CookieOptions) {
```

Comment:
Can we allow selectively choosing what is enabled and disabled? Maybe that means replacing `isDisabled` with something like 
```
interface StorageOptions {
  cookies: boolean,
  localStorage: boolean,
  inMemory: boolean
}
```
or 
```
interface Storage {
  cookie: Store
  localStorage: Store
  inMemory: Store
}
```

Just thinking of all the different permutations we support/want to support (just cookies, just localstorage, all of the above, nothing)

Reply:
Thinking of this more - instead of option 2, could even pass in an array of stores - we don't really need to know if they are cookies or localstorage or whatever.

Reply:
@chrisradek I think they generally only need one type of storage AFAIK (they don’t use all of them?) — so we just accept a single storage interface as an override (they could even directly pass in window.localStorage)

Reply:
FYI I think isDisabled as a configuration option is odd — I don’t think classes should be responsible for creating stub versions of themselves — I’d rather have that logic be moved up a level, seems a bit more idiomatic.